etl/expiry.py

```diff
@@ -15,7 +15,7 @@ def get_expiry_text(expiry, app_name, product_details):
 
     if expiry == "never" or expiry is None:
         return expiry
-    elif app_name == "firefox_desktop":
+    if type(expiry) == int and (app_name in ["firefox_desktop", "fenix", "firefox_ios"]):
```

Comment:
Does this mean we need to update it here when other apps switch to version based expiry?

Reply:
yep. Will file an issue to keep track of this.